.sample.env

```diff
@@ -4,10 +4,10 @@ GA_API_EMAIL_ADDRESS=xx
 GA_PRIVATE_KEY_FILENAME=xx
 
 FOG_PROVIDER=xxx
-RACKSPACE_USERNAME=xxx
-RACKSPACE_API_KEY=xxx
-FOG_DIRECTORY=xxx
-RACKSPACE_CDN_HOST=xxx
+AZURE_ASSETS_STORAGE_ACCOUNT_NAME=xxx
```

Comment:
I'm pretty sure we're going to actually go with `AZURE_ASSETS_STORAGE_BLOG_ACCOUNT_NAME` and `AZURE_ASSETS_STORAGE_BLOG_ACCOUNT_KEY` here. See https://github.com/moneyadviceservice/scripts/pull/585 and I'm doing a PR to add these variables to `/etc/mas/environment` in scripts.